client/src/components/layouts/Dashboard.js

```diff
@@ -4,21 +4,35 @@ import{NavLink as RouterNavLink} from 'react-router-dom';
 
 const Dashboard = () => {
     return (
-        <div>         
-            <Table responsive="md" borderless>
-                <tbody align="center" justifyContent="center">
-                    <tr height="200px">
-                        <td>Create New Quiz</td>
-                        <td>Past Quizzes</td>
-                        <td>Mailing List</td>
-                    </tr>
-                    <tr>
-                        <td>Manage Account</td>
-                        <td>Customer Service</td>
-                        <td>image placeholder</td>
-                    </tr>
-                </tbody>   
-            </Table>         
+        <div>    
+            <Container style={{marginTop:30, marginBottom:15}}>
+                <span class="mb-0 h3">Dashboard / Access your services</span>
+            </Container>  
+            <div className='dash-card-row'>
+                <div className='dash-card-cell'>
+                    <a href="#">Create Quiz</a> 
+                </div>
+                <div className='dash-card-cell'>
+                    <a href="#">Past Quizzes</a> 
+                </div>
+                <div className='dash-card-cell'>
+                    <a href="#"> Mailing List (TBD)</a> 
+                </div>
+            </div>
+            <div className='dash-card-row'>
+                <div className='dash-card-cell'>
+                    <a href="#">  Manage Account</a> 
+                </div>
+                <div className='dash-card-cell'>
+                    <a href="ContactUs"> Customer Service</a> 
+                </div>
+                <div className='dash-card-cell'>
+                    <a href="#"> <img src = {require("../../assets/logo_small_two.png")} className='img-fluid shadow-4'/></a> 
+                </div>
+                <div>
+                   All copyrights reserved, quizbanana.,llc 2022
```

Comment:
not important right now but it's possible to do 2022 as dynamic. please refer here https://updateyourfooter.com/

Reply:
I will do!